shaper_1187.xml

```diff
@@ -19,8 +19,8 @@ float shape = 0.0f;
 
 if (shapep < 1.0f && shapep > -1.0f) {
 	shape = 1.0f;
-} else if (shape < 0) {
-	shape = -1.0f / shape;
+} else if (shapep < 0) {
+	shape = -1.0f / shapep;
```

Comment:
The commit message says `division with 0` but that's wrong. `shape < 0` never is true so we drop to else for any number that is negative and feed this as exponent in a pow() operation. No fun.